easybuild/easyconfigs/a/ANIcalculator/ANIcalculator-1.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,46 @@
+# This installation only works for Linux distributions that provide libnsl.so.1 as part of the (g)libc package.
+easyblock = 'MakeCp'
+
+name = 'ANIcalculator'
+version = '1.0'
+
+homepage = 'https://ani.jgi.doe.gov/html/home.php'
+description = """This tool will calculate the bidirectional average nucleotide identity (gANI) and 
+Alignment Fraction (AF) between two genomes. Required input is the full path to the fna file 
+(nucleotide sequence of genes in fasta format) of each query genome. Either the rRNA and tRNA genes 
+can be excluded, or provided in a list with the -ignoreList option. This is necessary as the presence
+of tRNA and/or rRNA genes in the fna will artificially inflate the ANI."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://ani.jgi.doe.gov/download_files/']
+sources = ['%(name)s_v%(version_major)s.tgz']
+checksums = ['236596a9a204cbcad162fc66be3506b2530b1f48f4f84d9647ccec3ca7483a43']
+
+# A build dependency on binutils is required to pass the test suite.
```

Comment:
I think this comment can be removed, it can be confusing (at first I read it as saying that an `ANIcalculator` test suite required it)

(and I wouldn't say it's required to pass the easyconfigs test suite, the test suite tests for it because it is required when using GCCcore :) )

Reply:
Okay, I've removed it :-)

Reply:
ah, I only saw the discussion on slack now (facepalm...)